src/net.cpp

```diff
@@ -3145,13 +3145,16 @@ bool CConnman::Bind(const CService& addr_, unsigned int flags, NetPermissionFlag
     const CService addr{MaybeFlipIPv6toCJDNS(addr_)};
 
     bilingual_str strError;
-    if (!BindListenPort(addr, strError)) {
+    if (!BindAndStartListening(addr, strError)) {
+        LogPrintLevel(BCLog::NET, BCLog::Level::Error, "%s\n", strError.original);
         if ((flags & BF_REPORT_ERROR) && m_client_interface) {
             m_client_interface->ThreadSafeMessageBox(strError, "", CClientUIInterface::MSG_ERROR);
         }
         return false;
     }
 
+    LogPrintLevel(BCLog::NET, BCLog::Level::Info, "Bound to and listening at %s\n", addr.ToStringAddrPort());
```

Comment:
4c23aacc2b82083f96255963f543dc35fff28334: I'm inclined to say "listening **on** %s"

As in: we bind _to_ a port in order to listen _on_ it.

The linux manual for `listen` also uses "on", as in "listen for connections on a socket"
https://man7.org/linux/man-pages/man2/listen.2.html